### PR TITLE
CI : UI Tests (Ignore Sanity PPV2 in some paths)

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | 8.0.x
| Description?      | CI : UI Tests (Ignore Sanity PPV2 in some paths)
| Type?             | improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | N/A
| Fixed ticket?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

**Review by 0x346e3730 — DISMISSED** [review]

**Review by jolelievre — DISMISSED** [review]
Thanks @Progi1984

File: .github/workflows/sanity-productV2-skip.yml [hunk 1]
```diff
@@ -0,0 +1,25 @@
+name: UI tests
```
Comment (jolelievre) [inline .github/workflows/sanity-productV2-skip.yml]: @Progi1984 to match the CI on other branches and to also match the protected branch rules I think we should rename this one to:

Comment (jolelievre) [reply .github/workflows/sanity-productV2-skip.yml]:
```suggestion
name: UI tests Product V2
```

Comment (jolelievre) [reply .github/workflows/sanity-productV2-skip.yml]: And same in `sanity-productV2.yml` of course

**Comment by Progi1984** [comment]
Done @jolelievre 

Ping @0x346e3730 & @jolelievre  for review

**Review by nicosomb — APPROVED** [review]

**Review by mflasquin — APPROVED** [review]

**Review by 0x346e3730 — APPROVED** [review]

**Pull request merged**